Sources/Sentry/SentryOptions.m

```diff
@@ -51,6 +54,18 @@ - (void)validateOptions:(NSDictionary<NSString *, id> *)options
     }
     
     self.dsn = [[SentryDsn alloc] initWithString:[options valueForKey:@"dsn"] didFailWithError:error];
+
+    if (nil != [options objectForKey:@"debug"]) {
+        self.debug = [NSNumber numberWithBool:[[options objectForKey:@"debug"] boolValue]];
+    } else {
+        self.debug = @NO;
+    }
+
+    if ([self.debug isEqual:@YES])  {
+        self.logLevel = kSentryLogLevelVerbose;
+    } else {
+        self.logLevel = kSentryLogLevelError;
+    }
```

Comment:
Even if `logLevel` is deprecated now, it still wouldn't be considered now or am I wrong?
So if someone sets `logLevel` in the options it will always be `Error` or `Verbose` because `debug` overwrites it?

Reply:
Yes and no. 

logLevel isn't exposed at all when initializing (no matter if init complete Hub/Client a.s.o. or just inti `SentryOptions`). BUT you can change it later in SentryOptions, since it's exposed there.

This means...

This works (both calls have the same effect):
```
SentrySDK.start(options: [
    "dsn": "foo",
    "debug": false
])
SentrySDK.currentHub().getClient()?.options.logLevel = .verbose
```

And this doesn't work - has no effect:
```
SentrySDK.start(options: [
    "dsn": "foo",
    "logLevel": SentryLogLevel.verbose
])
```

maybe this overcomplicates things. but IMHO a nice way to hide the soon to be removed loglevel and still have access to it as long as it exists. also throws an immediate build error when users update from 5.x to the version that completely kicks loglevel.

Reply:
Gotcha

Reply:
yeah